Samurai/Modules/TippingModule.cs

```diff
@@ -1,20 +1,16 @@
 using System;
-using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using AcidChicken.Samurai.Assets;
+using AcidChicken.Samurai.Components;
+using AcidChicken.Samurai.Models;
+using AcidChicken.Samurai.Tasks;
```

Comment:
All of the local namespace reference instructions have to be included in the local namespace block.